### ISSUE TITLE
Add support for p11test testsuite

### ISSUE THREAD

**Issue by PeterHuewe** [tpm2-software/tpm2-pkcs11#526]
The OpenSC Project has a pkcs11 specific testsuite which can run against all kind of pkcs11 providers.
See https://github.com/OpenSC/OpenSC/tree/master/src/tests/p11test for Details.

Integration should be quite easy
```
git clone https://github.com/OpenSC/OpenSC.git
cd OpenSC
./bootstrap
./configure
make -j4
cd src/test/p11test/
./runtest.sh readonly /usr/lib/arm-linux-gnueabihf/pkcs11/libtpm2_pkcs11.so
```
And it actually shows atleast one failed testcase (ec_sign_size_test) (and quite a few error messages)

**Comment by williamcroberts** [comment]
@PeterHuewe what are the error messages? Is it related to #515?

**Comment by PeterHuewe** [comment]
No, it fails to validate some rsa signatures, but this does not fail the testsuite, just emits the messagge.
I'll run it again tomorrow and post the results.
